include/zephyr/kernel/internal/sched_priq.h

```diff
@@ -10,6 +10,10 @@
 #include <zephyr/sys/dlist.h>
 #include <zephyr/sys/rb.h>
 
+#define NBITS (8 * sizeof(uintptr_t))
+#define K_NUM_THRAD_PRIO (CONFIG_NUM_PREEMPT_PRIORITIES + CONFIG_NUM_COOP_PRIORITIES + 1)
+#define PRI_QUEUE_BITMAP_SIZE ((K_NUM_THRAD_PRIO + NBITS - 1) / NBITS)
+
```

Comment:
This (or something like it) should probably go into sched.c file, as Andy mentioned before.

Reply:
It can’t. Because the  _prio_q  will use this macro.
I already minimize the macros.